### PR TITLE
LICENSE and NOTICE

### PR DESCRIPTION
**Pull request by marcin-krolik**
License updated with derived work

### CODE REVIEW

File: NOTICE [hunk 1]
```diff
@@ -1,7 +1,5 @@
 snap-plugin-collector-docker
```
Comment (katarzyna-z) [inline NOTICE]: Here you should replace "snap-plugin-collector-docker" with "Intel Snap"

File: LICENSE [hunk 2]
```diff
@@ -175,3 +175,10 @@
       of your accepting any such warranty or additional liability.
 
    END OF TERMS AND CONDITIONS
+
+
+This product contains code from OCI/opencontainers, which is available
+under an Apache License 2.0. For details, see container/statistics.go, container/cgroupfs/blkio.go.
+
+This product contains code from Google cAdvisor, which is available
+uder an Apache License 2.0. For details, see container/fs/fs.go, container/fs/types.go, container/network/network.go, container/network/tcp.go
```
Comment (katarzyna-z) [inline LICENSE]: In mentioned files (container/statistics.go, container/cgroupfs/blkio.go, ontainer/fs/fs.go, container/fs/types.go, container/network/network.go, container/network/tcp.go) if there are original to Intel but with snippets from 3rd party components - files should have two Copyrights headers: Intel + 3rd component, see [this](https://github.com/intelsdi-x/snap-plugin-publisher-heapster/blob/master/exchange/exchange.go) example.

File: NOTICE (hunk 1 above)
Comment (marcin-krolik) [inline NOTICE]: done

File: LICENSE (hunk 2 above)
Comment (marcin-krolik) [inline LICENSE]: done

**Pull request merged**